mock/jest-mocks.ts

```diff
@@ -0,0 +1,21 @@
+import * as RawMocks from ".";
```

Comment:
I wanted to avoid duplicating source of truth for mocks (i.e. using arrow functions here), so I've decided to "reuse" existing mocks (which are left intact)